src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/normalize.cpp

```diff
@@ -64,7 +64,13 @@ class NormalizeL2LayerCPUTest : public testing::WithParamInterface<NormalizeL2La
         if (selectedType.empty()) {
             selectedType = getPrimitiveType();
         }
-        selectedType = makeSelectedTypeStr("unknown", inType);
+        // BF16 is not supported for NormalizeL2 on AVX2_VNNI_2 platforms
```

Comment:
On any system w/o HW BF16 support? Then more accurate condition is: `if (!with_cpu_x86_bfloat16() && inType == ElementType::bf16)`